package.json

```diff
@@ -28,25 +28,30 @@
     "tslint": "tslint \"app/**/*.ts\""
   },
   "dependencies": {
-    "@angular/core": "2.0.0-rc.5",
-    "@angular/common": "2.0.0-rc.5",
-    "@angular/compiler": "2.0.0-rc.5",
-    "@angular/http": "2.0.0-rc.5",
-    "@angular/platform-browser": "2.0.0-rc.5",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.5",
-    "@angular/platform-server": "2.0.0-rc.5",
-    "@angular/forms": "0.3.0",
-    "@angular/router": "3.0.0-rc.1",
+    "nativescript-angular": "0.5.0",
+    "tns-core-modules": "2.2.1",
+    "nativescript-intl": "^0.0.4",
+    "@angular/core": "2.0.0-rc.6",
+    "@angular/common": "2.0.0-rc.6",
+    "@angular/compiler": "2.0.0-rc.6",
+    "@angular/http": "2.0.0-rc.6",
+    "@angular/platform-browser": "2.0.0-rc.6",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.6",
+    "@angular/platform-server": "2.0.0-rc.6",
+    "@angular/forms": "2.0.0-rc.6",
+    "@angular/router": "3.0.0-rc.2",
+    "rxjs": "5.0.0-beta.11",
+    "zone.js": "^0.6.17",
+    "reflect-metadata": "^0.1.8",
+    "parse5": "1.3.2",
+    "punycode": "1.3.2",
+    "querystring": "0.2.0",
+    "url": "0.10.3",
```

Comment:
I’m also curious about the 4 dependencies above. Do they need to be explicitly in here?